examples/memory_agentic_chatbot_groq/README.md

```diff
@@ -0,0 +1,122 @@
+
+# Wise Agents Example: Memory Agentic Chatbot with GROQ api
+
+This guide walks you through running a practical example of a multi-agent system using Wise Agents. In this example, two agents (a web interface agent and an intelligent agent) are started, allowing you to experiment with agent communication and interaction in a simulated environment. This is the same descibed in [memory_agentic_chatbot](../memory_agentic_chatbot/), but instead of using a locla LLM it uses [https://groq.com/](https://groq.com/) for inference the model. Please obtain an API key from them and set env variable GROQ_API_KEY to start playing with this example
+
+
+## Example Overview
+
+The example consists of two main agents:
+
+1. **Web Interface Agent**: Simulates a web-based client for interacting with other agents.
+2. **Intelligent Agent**: Handles requests and provides intelligent responses based on memory and context.
+
+These agents are defined in YAML configuration files located in the `examples/memory_agentic_chatbot` directory.
+
+## Running the Example
+
+### Step 1: Clone the Repository
+
+If you haven't already, clone the Wise Agents repository from GitHub:
+
+```bash
+git clone https://github.com/wise-agents/wise-agents.git
+cd wise-agents
+```
+
+### Step 2: Configure and Start Redis
+
+In this step, we will set up Redis for agent context and registry.
+
+1. **Create a hidden directory `.wise-agents`** in the root of your project:
+
+   ```bash
+   mkdir .wise-agents
+   ```
+
+2. **Copy the Redis configuration file** as shown in the [`.wise-agents` directory](https://github.com/wise-agents/wise-agents/tree/main/.wise-agents) from the GitHub repo. Create a file named `redis-config.yaml` inside `.wise-agents`:
+
+   ```yaml
+   redis:
+     host: localhost
+     port: 6379
+   ```
+
+3. **Ensure Redis is installed and running**. You can start redis as a podman/docker image following instructions in [redis README.MD](../../redis/README.MD)
```

Comment:
s/following instructions in [redis README.MD]/following the instructions in the [Redis README.MD]